app/Jobs/ImportRockTheVoteRecord.php

```diff
@@ -310,12 +312,16 @@ private function createUser($record)
      */
     private function getVoterRegistrationStatusChange($currentStatus, $newStatus)
     {
+        // List includes status values expected from RTV as well as
+        // values potentially assigned from within Northstar.
         $statusHierarchy = [
             'uncertain',
             'ineligible',
+            'unregistered',
             'confirmed',
             'register-OVR',
             'register-form',
+            'registration_complete',
```

Comment:
I thought the `register-OVR` and `register-form` got compacted into `registration_complete` on users in Northstar?

Reply:
They totally do!https://github.com/DoSomething/chompy/blob/e5a36f7a218a67074ca7209ede1344b3edd6d098/app/Jobs/ImportRockTheVoteRecord.php#L48 We do use the `register-OVR/form` for the Post though https://github.com/DoSomething/chompy/blob/e5a36f7a218a67074ca7209ede1344b3edd6d098/app/Jobs/ImportRockTheVoteRecord.php#L56
So I think this way we can use this validation across Post and profile updates?

Reply:
AH YES! This is what I expected, thank you!!